src/core/src/op/paged_attention.cpp

```diff
@@ -147,40 +147,53 @@ void PagedAttentionExtension::validate_and_infer_types() {
                           get_input_element_type(12),
                           ".");
 
-    if (get_input_size() == 16) {
+    NODE_VALIDATION_CHECK(this,
```

Comment:
too much code duplication in this file. It's getting hard to review and maintain.
can we re-fine we pshape and types check this way?
input_check(idx, input_name, {list of ranks}, {list of types});

e.g.
input_check(13, "score_aggregation_window", {dynamic, 0, 1}, {dynamic, element::13});

Reply:
I suppose that this would be a task for the code owner/maintainer of the file and/or component.